src/dvc_objects/db.py

```diff
@@ -142,6 +142,13 @@ def add(
                 callback=Callback.as_callback(cb),
             )
 
+    def delete(self, oid: str):
```

Comment:
Should this be `remove()`?

Reply:
We also have `dvc-data rm` already.

Reply:
rm sounds file-centric, while `delete` feels more general. This is handling oids so I've used `delete` for now as an anti-add.